SimPPS/PPSPixelDigiProducer/BuildFile.xml

```diff
@@ -0,0 +1,21 @@
+<use   name="SimDataFormats/TrackingHit"/>
+<use   name="SimGeneral/NoiseGenerators"/>
+<use   name="FWCore/ParameterSet"/>
+<use   name="FWCore/Framework"/>
+<use   name="DataFormats/Common"/>
+<use   name="DataFormats/CTPPSDigi"/>
+<use   name="SimDataFormats/CrossingFrame"/>
+<use   name="SimTracker/Common"/>
+<use   name="Geometry/VeryForwardGeometry"/>
+<use   name="Geometry/VeryForwardGeometryBuilder"/>
+<use   name="clhep"/>
+<use   name="Mixing/Base"/>
+<use   name="gsl"/>
+<use   name="boost"/>
+<use   name="geant4"/>
```

Comment:
@mundim , you not need dependence on Geant4 (also dependences on other unused libraries better to remove)

Reply:
done. In fact, many other unneeded libraries were removed...